### DISCUSSION TITLE
How can I connect a Flutter app to my existing Filament Laravel backend?

### DISCUSSION THREAD

**Discussion by hamadmh** [filamentphp/filament#15184] [Help]
### Package

Other

### Package Version

v2.17.43

### How can we help you?

Hello,
I'm new to using Filament in Laravel, and I’ve built a web application using it. I now want to connect this backend to a Flutter app by creating an API. Since I’ve never created or connected an API with Filament before, I’m unsure where to start.

Could someone guide me on the best approach to expose API endpoints from my current Filament source code for Flutter? Do I need to make changes to my existing codebase, or is there a built-in way to achieve this?

Your help would be greatly appreciated!

**Comment by leandrocfe** [comment]
Maybe [this](https://filamentphp.com/plugins/rupadana-api-service)?
